modules/platforms/dotnet/Apache.Ignite.Core.Tests/Binary/BinaryDateTimeTest.cs

```diff
@@ -134,10 +133,22 @@ public void TestCustomTimestampConverter()
 
             AssertTimestampField<DateTimeObj2>((o, d) => o.Value = d, o => o.Value, "Value");
 
-            var ex = Assert.Throws<BinaryObjectException>(() => binary.ToBinary<IBinaryObject>(new DateTime(1997, 8, 29)), 
-                "Converter Error!");
+            var dt1 = new DateTime(1997, 8, 29, 0, 0, 0, DateTimeKind.Utc);
+
+            var ex = Assert.Throws<BinaryObjectException>(() => binary.ToBinary<DateTime>(dt1));
+            Assert.AreEqual("ToJavaTicks Error!", ex.Message);
```

Comment:
Let's extract a constant to avoid repeating `"ToJavaTicks Error!"` value.

Reply:
Fixed.